### DISCUSSION TITLE
Discord server, Any eta?

### DISCUSSION THREAD

**Discussion by Mafy-17** [theomenden/TheOmenDen.Vanadium#17] [Q&A]
do youi have a discord server and do you have an relative estimate of time till first working public alpha build ?

**Comment by AluTheCrow** [comment]
> do youi have a discord server and do you have an relative estimate of time till first working public alpha build ?

Discord server: https://discord.gg/GWshtCqEx4

As far as public build goes, I'm trying to get it out ASAP, got some mixins to fix, and as soon as I can get those down, I'll release a build here, and on Modrinth. Hoping sometime between 14/08 and 17/08 - depending on how much work has me hosed.
